src/components/section/index.js

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
 import styles from './style.css';
+import classNames from 'classnames';
 
-export default class Section extends React.Component {
-  render () {
-    return (
-      <section className={styles.section}>
-        {this.props.children}
-      </section>
-    );
-  }
+export default function Section (props) {
+  const sectionClass = classNames(props.className, styles.section);
```

Comment:
I don't know if this is the ideal pattern for components. Essentially it allows all of the style properties to be changed out from under the component. It also leads to ambiguity of how styles are applied and what order they are applied. The same can be achieved by wrapping components in another element with the intended styles. Thoughts?